1-HT-OOP/src/Shape.ts

```diff
@@ -1,3 +1,31 @@
+import { Point } from "./Point";
+
 export abstract class Shape {
+    protected color:string;
+    protected filled:boolean;
+    protected points:Point[];
+    
+    
+    constructor(points?:Point[],color?:string,filled?:boolean);
+    constructor(points:Point[]=[new Point(4,5),new Point(6,8),new Point(6,10)],
```

Comment:
Don't hardcode the points from here, it is passed as a parameter from the object creation.